mmeval/metrics/utils/evaluate_semantic_instance.py

```diff
@@ -0,0 +1,346 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# adapted from https://github.com/ScanNet/ScanNet/blob/master/BenchmarkScripts/3d_evaluation/evaluate_semantic_instance.py # noqa
+import numpy as np
+from copy import deepcopy
+
+from . import util_3d
```

Comment:
This script seems adapted from `https://github.com/ScanNet/ScanNet`.
Copy it as it is and put it in `mmeval/metrics/_vendor`

Reply:
moved